src/queries/enum_repr_int_changed.ron

```diff
@@ -59,7 +70,8 @@ SemverQuery(
     }"#,
     arguments: {
         "public": "public",
-        "repr_regex": "#\\[repr\\([ui]\\d+|size\\)\\]",
+        "repr": "repr",
+        "repr_regex": "[ui]\\d+|size",
```

Comment:
```suggestion
        "repr_regex": "[ui](\\d+|size)",
```
I believe that in regex, the alternation (`|`) operator has the lowest precedence. If I'm correct on that then this regex was being interpreted as "either 'size', or 'u/i' followed by digits" which is not correct.

Consider adding test cases for `usize/isize` to verify whether this bug was actually present, and protect against such a regression in the future. Regex is tricky and non-obvious so even if we avoid this bug right now, it could happen again in the future. Better safe than sorry so we should have a test regardless.

Reply:
Actually we already had tests for these cases, but I agree that we add the brackets around `|` operator arguments for easier human-parsing and future-proofing.